src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -57,6 +57,14 @@
       "The connection URI as supported by the official drivers. "
           + "eg: ``mongodb://user@pass@locahost/``.";
 
+  public static final String JSON_FORMAT = "json.format";
+  private static final String JSON_DOC =
+      "This will provide in which type of JSON the output will be, we will have 3 type of mods : "
```

Comment:
To be adapted/broken into several lines:
```suggestion
      "The output mode of the ``JSONWriter``. The accepted values are ``canonical (strict?)`` (Legacy representation. Though now deprecated, this is still the default mode when writing JSON in order to avoid breaking backward compatibility.) and ``relaxed`` (Relaxed representation that loses type information for BSON numeric types and uses a more human-readable representation of BSON dates.)."
```